packages/core/upload/server/graphql.js

```diff
@@ -184,6 +184,16 @@ module.exports = ({ strapi }) => {
       },
     });
 
-    return { types: [fileInfoInputType, mutations] };
+    return {
```

Comment:
WHat's the link with this PR ?

Reply:
Because it is linked to the removal of restricted relations from content APIs. We need to set the right scopes (those you can chose in UP permissions page) for the resolvers